AI/CMakeLists.txt

```diff
@@ -45,9 +48,11 @@ endif()
 #######################################
 
 add_subdirectory(BattleAI)
-add_subdirectory(StupidAI)
-add_subdirectory(EmptyAI)
 add_subdirectory(VCAI)
+if(NOT ENABLE_STATIC_AI_LIBS)
+	add_subdirectory(StupidAI)
+	add_subdirectory(EmptyAI)
+endif()
```

Comment:
Why not allow this libraries for Android as well?

Reply:
because static AI approach hardcodes selecting battle AI, see `CGameInterface.cpp`

Reply:
Why not add StupidAI & EmptyAI in hardcoded list in CGameInterface.cpp?

Reply:
this would require adjusting the java launcher as well. I wouldn't put effort into the java launcher at all (unless strongly required) as porting Qt launcher is more important.

feel free to open such issue / work on it if you wish :)

Reply:
It won't *require* adjusting java launcher. We can change CGameInterface but keep only NKAI & VCAI available to user.

Mostly want to avoid potential issues in future, e.g. if Android gets Qt Launcher where StupidAI can be selected and we forget about non-functioning StupidAI in Android.

Anyway, not insisting, feel free to ignore.

Reply:
oh, I got the point, makes sense! User will also be able to change AI by editing the json settings.